### PR TITLE
Update /uasd, /ceph, /pricing/infra

### LINKED ISSUE #8359: Update /legal/ubuntu-advantage-service-terms
Wait for @lechsandecki +1

- [Brief](https://docs.google.com/document/d/1N-PEWFsHf20pzInRzdD0RR0t3752hpDCFCdl4rgASGs/edit)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update Ubuntu Advantage service description (/uasd), /ceph, /pricing/infra

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/uasd
    - http://0.0.0.0:8001/ceph
    - http://0.0.0.0:8001/pricing/infra
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to and resolve comments in the copy docs: [/uasd](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit#), [/ceph](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit?ts=5fc77c00#heading=h.zfsh8iinf6jd), [/pricing/infra](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit?ts=5fc77c00#heading=h.zfsh8iinf6jd)

## Issue / Card

Fixes #8359

### CODE REVIEW

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
On the demo, the _Software-defined storage - Ceph and Swift_ on `/pricing/infra` has different numbers to the [copy doc](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit?ts=5fc77c00#heading=h.zfsh8iinf6jd) (48 instead of 72)

image

File: templates/legal/ubuntu-advantage-service-description/index.md [hunk 1]
```diff
@@ -9,9 +9,8 @@ markdown_includes:
 
 # Ubuntu Advantage service description
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/legal/ubuntu-advantage-service-description/index.md:10]: The url and the h1 are correct, the doc is used in other things, so it has a different h1 there

[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/legal/ubuntu-advantage-service-description/index.md:10]: good catch, I didn't see the delete inside the addition.

[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/legal/ubuntu-advantage-service-description/index.md:10]: @bethcollins92 ok, fixed up those 2.

**Review by bethcollins92 — APPROVED** [review]
Thanks, all LGTM! 👍

**Pull request merged**